docs/processors/building-standalone-processors.mdx

```diff
@@ -0,0 +1,135 @@
+---
+title: 'Building Standalone Processors'
```

Comment:
I'd go a different route and do something like this:
```
processors
├── ... # Other pages
└── standalone
    ├── index.mdx    # General info
    │                # about how to use
    │                # standalone processors.
    ├── building.mdx # This page - how
    │                # to build your own
    │                # standalone processor.
    └── wasm.mdx     # The page I'm working
                     # on - how standalone
                     # processors work under
                     # the hood.
```

In the menu it would then look something like:
```
Standalone processors >
  How to build a standalone processor
  How it works (WASM)
```

The naming of files and pages can be adjusted, I'm mostly proposing the nesting structure.

Reply:
I like the nesting structure too

Reply:
done